CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## v2.2.6 14.02.2020
```

Comment:
Given the changes this should be at least 2.3 release, preferably a major version release.

Reply:
We haven't made any backwards-breaking API changes so a major release wouldn't be appropriate for semver but 2.3 sounds good to me